configs/mediawiki.php

```diff
@@ -23,6 +23,8 @@
 session_save_path( isset($hwConfig['session_save_path']) ? $hwConfig['session_save_path'] : $IP . '../../tmp/sessions' );
 
 # Load Hitchwiki Config
+# TODO https://secure.php.net/manual/en/book.yaml.php
+#$hwConfig = eval 'cat ../configs/settings.ini | shyaml get-value mapping';
 $hwConfig = parse_ini_file("settings.ini", true);
```

Comment:
So something like this here should work:
```php
# Load Hitchwiki Config
# Paths relative to `configs/mediawiki.php`
require_once '../public/mustangostang/spyc/spyc.php';
$hwConfig = spyc_load_file('settings.yaml');
```

...just need to check in what structure this loads data from the file, so that we can modify references like `$hwConfig['spam']['spamprotection']` accordingly. Just throwing `die(print_r($hw_config,true));` after these lines shows it.

Reply:
what about installing it into `scripts/vendor/spyc`, or `scripts/vendor/mustangostang/spyc` (easier to .gitignore)?